README.md

```diff
@@ -0,0 +1,3 @@
+# Conway's game of life
+Yet another Conway's game of life implementation
+
```

Comment:
You should add instructions on how to clone the repo and run the code

Reply:
Will do it in next PR